src/main/java/net/sf/jabref/importer/ImportMenuItem.java

```diff
@@ -219,34 +224,66 @@ public void update() {
                             }
                         }
 
+                        mainLoop:
```

Comment:
Hmm, I simply stole that piece of code from another JabRef source file... But, yes, probably.